Demo/Demo/SwiftUIComponents/SwiftUIPaymentButtonDemo.swift

```diff
@@ -111,6 +111,7 @@ struct SwiftUIPaymentButtonDemo: View {
                         size: selectedSize
                     )
                     .id(buttonID)
+                    .frame(maxWidth: .infinity)
```

Comment:
Are these needed? I removed them and tested in canvas and sim and seem to be okay without these changes.

Reply:
They're not needed but I have them there so that people, internally and externally, who aren't familiar with SwiftUI know that you can set the button widths. Pre-PR, this part was broken and you couldn't customize the width in SwiftUI. If it feels redundant and unnecessary, I can remove it

Reply:
Maybe better to remove them if they're not necessary.